content/en/docs/use-cases/bootstrapping/index.md

```diff
@@ -42,23 +42,23 @@ To support the [autofill UI](/docs/reference/terms/#autofill-ui) for passkeys, m
       const available = await PublicKeyCredential.isConditionalMediationAvailable();
 
       if (available) {
-         // Query your server for options for `navigator.credentials.get()`
         try {
+          // 2.1) Retrieve authentication options for
```

Comment:
Are these numbers supposed to be a reference to something? They read like spec references as-is, and if they're not I'm not sure why they use sub-numbering instead of just "1:", "2:", etc...

Reply:
This is indeed not yet very consistent, will remove. 👍